provider/datagen/src/baked_exporter.rs

```diff
@@ -450,10 +469,12 @@ impl DataExporter for BakedExporter {
                                 req: icu_provider::DataRequest,
                             ) -> Result<icu_provider::DataResponse<#marker>, icu_provider::DataError> {
                                 let locale = req.locale;
+                                #[allow(unused_mut)] // mutable for fallback
```

Comment:
q: where does this actually get mutated?

Reply:
The fallback loop set the resolved locale before breaking.

Reply:
ah, right. ty